src/languages/stan.js

```diff
@@ -504,16 +345,38 @@ export default function(hljs) {
       $pattern: hljs.IDENT_RE,
       title: BLOCKS,
       keyword: STATEMENTS.concat(VAR_TYPES).concat(SPECIAL_FUNCTIONS),
-      built_in: FUNCTIONS
+      built_in: FUNCTIONS.concat(DISTRIBUTIONS).concat(expanded),
```

Comment:
Now lets find a better named for `expanded`... I only was showing you the idea in the example, not the naming.

Reply:
Will update to DISTRIBUTIONS_EXPANDED